CHANGES_NEXT_RELEASE

```diff
@@ -5,3 +5,4 @@
 - Fix: Avoid faster libcoap response: libcoap is not waiting long enough for lightweight-iotagent and lwm2m-node-lib to send the response (#104)
 - Fix: cannot handle update registration from a Wakaama client (#106)
 - Update COAP librery to version 0.21.0
+- Update MongoDB and Mongoose version to 2.2.35 and 4.13.12 respectively
```

Comment:
Mongo has been set to 4.4.3 in this PR... 4.13.12 in CNR is not correct.

In fact, if this PR is going back to the situation previous to #129 I think the right line should be just:

```
- Update MongoDB to 2.2.35
```

Reply:
Fixed with 0b6c08ad873b55f263a7a528fcb1fac2982bf1ab